Occasional segmentation fault when running `update_preference_docs.py`

## 🐛 Bug

The `generate_images()` function on `docs/_scripts/update_preference_docs.py` occasionally results in a Segmentation Fault. This prevents the docs build.

## To Reproduce

Steps to reproduce the behavior:

1. Run `make docs` from the root napari folder.

Output (for approximately 9 out of 10 runs)

```
$ make docs
rm -rf docs/_build/
find docs/api ! -name 'index.rst' -type f -exec rm -f {} +
pip install -qr docs/requirements.txt
python docs/_scripts/update_preference_docs.py
make: *** [Makefile:7: docs] Segmentation fault (core dumped)
```

After this works once, generating all the images, if I comment the `generate_images()` function call the docs are built successfully with no other issues.

## Expected behavior

The docs are correctly built.

## Environment

napari: 0.4.13.dev129+g225d1548.d20211213
Platform: Linux-5.14.18-1-MANJARO-x86_64-with-glibc2.33
System: Manjaro Linux
Python: 3.9.7 | packaged by conda-forge | (default, Sep 29 2021, 19:23:11) [GCC 9.4.0]
Qt: 5.15.2
PyQt5: 5.15.6
NumPy: 1.21.4
SciPy: 1.7.3
Dask: 2021.11.2
VisPy: 0.9.4

OpenGL:
- GL version: 4.6 (Compatibility Profile) Mesa 21.2.5
- MAX_TEXTURE_SIZE: 16384

Screens:
- screen 1: resolution 1920x1080, scale 1.0
- screen 2: resolution 1920x1080, scale 1.0

Plugins:
- console: 0.0.4
- scikit-image: 0.4.13.dev129+g225d1548.d20211213
- svg: 0.1.5

--- a/napari/_qt/qt_event_loop.py
+++ b/napari/_qt/qt_event_loop.py
@@ -156,7 +156,7 @@ def get_app(
 
         # Intercept tooltip events in order to convert all text to rich text
         # to allow for text wrapping of tooltips
-        app.installEventFilter(QtToolTipEventFilter(app))
+        app.installEventFilter(QtToolTipEventFilter())
 
     if not _ipython_has_eventloop():
         notification_manager.notification_ready.connect(

